docker/README.md

```diff
@@ -1,93 +1,154 @@
-# Trino Gateway Docker Image
+# Trino Gateway Docker image
 
-## About the Container
+The Docker image of Trino Gateway is designed for the following use cases:
 
-This Docker image is designed to:
+* Manual usage in front of Trino clusters by mounting in a configuration file
+* Automated usage with an orchestration system like Kubernetes to simplify
+  deployment
 
-* be spun up in front of Trino clusters by mounting in a configuration file
-* simplify deployment into an orchestration system
+## Build requirements
 
-## Quickstart
-
-### Dependencies
-
-This docker build process requires:
+This docker build process requires the following software:
 
 * [Docker Compose V2](https://docs.docker.com/compose/)
 * jq
```

Comment:
how about adding java 17, mvn?

Reply:
No .. thats in the development docs .. we will ultimately merge them but dont want to duplicate now.